doc/examples/histology_resdnn.py

```diff
@@ -72,10 +71,10 @@
 resdnn_model.load_model_weights(fetch_model_weights_path)
 predicted_sh = resdnn_model.fit(data, gtab, mask=mask)
 
-nib.save(nib.Nifti1Image(predicted_sh, dwi_img.affine), 'predicted_sh.nii.gz')
+save_nifti('predicted_sh.nii.gz', predicted_sh, affine)
```

Comment:
I really don't like that function, I would rather avoid it. I prefer using Nibabel directly.